portafly/src/components/Root.tsx

```diff
@@ -53,13 +61,35 @@ const Root: React.FunctionComponent = ({ children }) => {
     }),
     [history]
   )
+
+  const pageToolbar = (
```

Comment:
Can we move this example to Overview page (or an "example" page)... Or are you planning to remove this before merge?

Reply:
I'll create an example page